server/src/controllers/email.js

```diff
@@ -0,0 +1,21 @@
+const express = require('express');
+const router = express.Router();
+const db = global.myDb;
+
+router.post('/', async (req, res) => {
+    let body = req.body;
+    try {
+        let result = await db.email.insertOne(body.email);
+        console.log(result);
+        res.sendStatus(200);
+    } catch (e) {
+        if (/already\sexists/.test(e.detail)) {
```

Comment:
Is this a typo? sexists vs exists? ex:

if (/already\exists/.test(e.detail)){

Reply:
\s is the character for space

Reply:
so what I'm testing for is "already exists" but specficially saying a \s (space) between